SCSH_3C_iOS/Networking/Moya/MoyaApiInterface/Distributor/DistributorApiInterface.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  DistributorApiInterface.swift
+//  Networking
+//
+//  Created by 辜敬閎 on 2023/12/4.
+//
+
+import Foundation
+import Moya
+
+public class DistributorApiInterface: MoyaApiInterfaceType {
+    public typealias OutputModel = ApiResponse<[Distributor]>
+    public typealias OutputError = ApiError
+
+    public init(limit: Int? = nil) {
+        if let limit = limit {
+            task = .requestParameters(parameters: ["limit": limit], encoding: URLEncoding.default)
+        } else {
+            task = .requestPlain
+        }
+    }
+    
+    public var baseURL: URL = URL(string: NetworkConstants.httpUrlScheme + NetworkConstants.localHost8080)!
+    public var path: String = "\(NetworkConstants.apiRootPath)/\(NetworkConstants.apiVersion)/distributors"
+    public var method: Moya.Method = .get
```

Comment:
I am not sure, but it seems that `let` instead of `var` is more suitable here for me.

Reply:
you're right, I miss it. Constant should be used instead.

Reply:
https://github.com/No-57/SCSH_3C_iOS/pull/51/commits/f8517f95feca584884a337123e8c579ea563eb98